frontend/services/src/maps/mapsService.ts

```diff
@@ -32,7 +32,7 @@ export const initializeAutocomplete = (inputRef: React.RefObject<HTMLInputElemen
     return null;
   }
   const options = {
-    types: ["address"],
+    types: ["geocode", "establishment"],
```

Comment:
hey danny, thanks for this - is it possible to make it so that it shows the place not the address? think from consumer pov it would be a lot nicer this way

Reply:
yeah but in event card can we show the place or nah?